OOps/ugens3.c

```diff
@@ -195,11 +251,28 @@ int32_t foscili(CSOUND *csound, FOSC *p)
         v1 = *ftab++;
         ar[n] = (v1 + (*ftab - v1) * fract) * amp;
         cphs += cinc;
+        } else {
+        mphsf = PHMOD1(mphsf);
```

Comment:
Awkward indentation

Reply:
yep, forgot to tidy up formatting.  Fixed now.